# Vacuum gripper control

# Enable/Disable gripper suction
bool enable

---
bool success
